src/code/audio/audio_data.c

```diff
@@ -0,0 +1,895 @@
+
+    // 8th Harmonic
+    0,      0,      32767,  32767,  0,      0,      -32767, -32767,
+    0,      0,      32767,  32767,  0,      0,      -32767, -32767,
+    0,      0,      32767,  32767,  0,      0,      -32767, -32767,
+    0,      0,      32767,  32767,  0,      0,      -32767, -32767,
+    0,      0,      32767,  32767,  0,      0,      -32767, -32767,
+    0,      0,      32767,  32767,  0,      0,      -32767, -32767,
+    0,      0,      32767,  32767,  0,      0,      -32767, -32767,
+    0,      0,      32767,  32767,  0,      0,      -32767, -32767,
+};
+
+s16 gWhiteNoiseSample[] = {
+    // 1st Harmonic
+    0,      -25689, -25791, 27803,  -27568, -21030, 22174,  6298,
+    27071,  -18531, 28649,  2284,   3380,   6890,   -12682, -21114,
+    10000,  -24520, 32296,  12248,  15096,  15354,  -12021, -31545,
+    -12929, 6388,   -11064, 30456,  -30316, -21999, 29691,  27649,
+    0,      -27649, -29691, 21999,  30316,  -30457, 11064,  -6387,
+    12929,  31544,  12021,  -15353, -15096, -12249, -32296, 24521,
+    -10000, 21113,  12682,  -6889,  -3380,  -2285,  -28649, 18532,
+    -27071, -6299,  -22174, 21031,  27568,  -27804, 25791,  25690,
+
+    // 2nd Harmonic
+    0,      -25791, -27568, 22174,  27071,  28649,  3380,   -12682,
+    10000,  32296,  15096,  -12021, -12929, -11064, -30316, 29691,
+    0,      -29691, 30316,  11064,  12929,  12021,  -15096, -32296,
+    -10000, 12682,  -3380,  -28649, -27071, -22174, 27568,  25791,
+    0,      -25791, -27568, 22174,  27071,  28649,  3380,   -12682,
+    10000,  32296,  15096,  -12021, -12929, -11064, -30316, 29691,
+    0,      -29691, 30316,  11064,  12929,  12021,  -15096, -32296,
+    -10000, 12682,  -3380,  -28649, -27071, -22174, 27568,  25791,
+
+    // 4th Harmonic
+    0,      -27568, 27071,  3380,   10000,  15096,  -12929, -30316,
+    0,      30316,  12929,  -15096, -10000, -3380,  -27071, 27568,
+    0,      -27568, 27071,  3380,   10000,  15096,  -12929, -30316,
+    0,      30316,  12929,  -15096, -10000, -3380,  -27071, 27568,
+    0,      -27568, 27071,  3380,   10000,  15096,  -12929, -30316,
+    0,      30316,  12929,  -15096, -10000, -3380,  -27071, 27568,
+    0,      -27568, 27071,  3380,   10000,  15096,  -12929, -30316,
+    0,      30316,  12929,  -15096, -10000, -3380,  -27071, 27568,
+
+    // 8th Harmonic
+    0,      27071,  10000,  -12929, 0,      12929,  -10000, -27071,
+    0,      27071,  10000,  -12929, 0,      12929,  -10000, -27071,
+    0,      27071,  10000,  -12929, 0,      12929,  -10000, -27071,
+    0,      27071,  10000,  -12929, 0,      12929,  -10000, -27071,
+    0,      27071,  10000,  -12929, 0,      12929,  -10000, -27071,
+    0,      27071,  10000,  -12929, 0,      12929,  -10000, -27071,
+    0,      27071,  10000,  -12929, 0,      12929,  -10000, -27071,
+    0,      27071,  10000,  -12929, 0,      12929,  -10000, -27071,
+};
+
+// Sine White Noise?
+s16 D_801D4790[] = {
```

Comment:
```suggestion
s16 gSineWhiteNoiseSample[] = {
```

Reply:
This data is the third one from the bottom. There are proper technical ways to describe noise, and the description `sine white noise` was just based on observation, and not properly characterizing it. It's hard to say if it's even suppose to be a sine wave or if it's just the type of noise (it's mirrored and flipped vertically from the left and right halfs of the noise, so that may have made it artificially look like a sine wave).

I do think there's potential to look into this more before given it a proper name, so I think I'd prefer to leave it unnamed just for now, thoughts?

![image](https://user-images.githubusercontent.com/47598039/144945888-8115d601-c064-4fb6-93f0-07ffb097f943.png)

Reply:
I don't have a better name for 85 either (just called it noise and havent found a use for it), it def looks like it was created with purpose, maybe in an attempt to create a wave used by a specific chip or legacy chip like NES or something, but have we actually found a place where it gets used? I never found a place in OOT or MM where these chiptunes get used, and assumed since this part of the engine existed in SM64 I assumed it was very old and intended to be used while sm64 was in development. I would have probably called it `UnknownNoiseSample85`.

Reply:
Alright, so I went down the rabbit hole of OoT (same principles would apply to MM) and I parsed through how these wave samples are used.

They're set by setting the instrument id, but with a value >= 128:
`&gWaveSamples[waveId - 128][sampleCountIndex * 64]`
This is in `Audio_BuildSyntheticWave`

waveId if you trace it back is set by `instOrWave` (instrument or Wave Id, instrument for < 128, Wave Id for >= 128)

This is read by the sequences in audio_seqplayer in commands `layer_setinstr` and `chan_setinstr` for example.

I took a look inside the sound effects sequence from OoT that we're currently working on. This specific unknown noise sample would be set by calling `layer_setinstr 133`, since 133 - 128 = 5 (index 5 of gWaveSamples).

There are 2 instances of `layer_setinstr 133` and 1 instance of `chan_setinstr 133`. Tracing those back, those sfx's are:
`NA_SE_EV_WATER_CONVECTION`
`NA_SE_EV_STONE_LAUNCH`
`NA_SE_EN_BIMOS_LAZER_GND`
(If I read the tables right)

Keeping in mind this is all for OoT. MM sequences will be extracted soon (TM)